src/uct/api/v2/uct_v2.h

```diff
@@ -473,6 +473,36 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief UCT endpoint connected by @ref uct_ep_connect_to_ep_v2 parameters
+ * field mask.
+ *
+ * The enumeration allows specifying which fields in @ref
+ * uct_ep_connect_to_ep_params_t are present, for backward compatibility
+ * support.
+ */
+enum uct_ep_connect_to_ep_params_field {
+    /* No fields are set */
+    UCT_EP_CONNECT_TO_EP_PARAM_FIELD_EMPTY = 0
```

Comment:
can we remove it?

Reply:
no: empty enum is not allowed :(

Reply:
so IMO better not define the enum, to not put things in H file that we would have to remove

Reply:
ok, removed